custom/opm/beneficiary.py

```diff
@@ -564,6 +564,24 @@ def weight_grade_normal(self):
                     return self.child_age/12
                 return False
 
+    def weight_grade_status(self, status):
+        if self.status == 'pregnant':
+            return False
+        if self.child_index == 1:
+            form_prop = 'interpret_grade'
+        else:
+            form_prop = 'interpret_grade_{}'.format(self.child_index)
+
+        forms = self.filtered_forms(CHILDREN_FORMS, 1)
+        if len(forms) == 0:
+            return False
```

Comment:
We are counting where the status is one of `['MAM', 'SAM', 'normal']`. So if there are no forms (may be because VHND didn't happen), by returning `False` we are not counting it. This doesn't falsely say a child case is not of `normal` weight since there are no forms, it just doesn't count this child case into number of children with Normal weight. That sounds correct?